stonesoup/tests/test_plotter.py

```diff
@@ -209,3 +209,12 @@ def test_plot_complex_uncertainty():
                                          "eignevalues or eigenvectors"):
 
         plotter.plot_tracks(track, mapping=[0, 1], uncertainty=True)
+
```

Comment:
Could manufacture some data and add more tests to cover loops in code where each piece data is looped through. Could also test _resize() at the same time.